src/openMVG/numeric/poly.h

```diff
@@ -22,7 +22,7 @@ namespace openMVG
 /**
 * @brief Solve roots of a cubic polynomial
 * \f$ x^3 + a x^2 + b x + c = 0 \f$
-* @param a Coefficient of cubic parameter
+* @param a Coefficient of quadratic parameter
```

Comment:
This a a cubic polynomial

Reply:
@pmoulon 
thank you for the comment.

a is the coefficient of x^2, which is a quadratic term. Shouldn't a be the quadratic
parameter?

Reply:
You're right. I'm sorry. I read to fast ;-)